main.go

```diff
@@ -94,9 +103,41 @@ func main() {
 		dief("looking for test data: %v\n", err)
 	}
 	for _, name := range remove {
-		if err := os.RemoveAll(name); err != nil {
-			dief("removing %v: %v\n", name, err)
+		err := osAwareRemoveAll(name);
+		if err != nil {
+			failedToRemove[name] = err
+		}
+	}
+	
+	if len(failedToRemove) > 0 {
+		fmt.Fprintf(os.Stderr, "\n# Failed to remove below item, please remove manually\n")
+		for name, err := range failedToRemove {
+			fmt.Fprintf(os.Stderr, "	%v: %v\n", name, err)
+		}
+	}
+}
+
+func osAwareRemoveAll(path string) error {
+	fileInfo, err := os.Stat(path)
+	if err != nil && os.IsNotExist(err) {
+		// skip if file not existed
+		return nil;
+	} else if err != nil {
```

Comment:
There's no reason for the "else" here because the previous statement returns. Just write

```
if os.IsNotExist(err) {
  return nil
}
if err != nil {
  return err
}
```